yt_dlp/extractor/zattoo.py

```diff
@@ -122,7 +133,24 @@ def _extract_cid_and_video_info(self, video_id):
 
         return cid, info_dict
 
-    def _extract_formats(self, cid, video_id, record_id=None, is_live=False):
+    def _extract_ondemand_info(self, ondemand_id):
+        data = self._download_json(
+            '%s/zapi/vod/movies/%s' % (self._host_url(), ondemand_id),
+            ondemand_id, 'Downloading ondemand information'
+        )
+        info_dict = {
+            'id': ondemand_id,
+            'title': data['title'],
```

Comment:
```suggestion
            'title': data.get('title'),
```
due to recent core changes, title is now non-fatal